.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "err-backend-gitter"]
 	path = err-backend-gitter
-	url = git@github.com:errbotio/err-backend-gitter.git
+	url = https://github.com/coala/corobo.git
```

Comment:
In travis, ssh is not public access, public access can be done through http url only. See https://docs.travis-ci.com/user/common-build-problems/#Git-cannot-clone-my-Submodules

I edited the `.gitmodules` file manually, is that a problem? 

However, this can be the problem here ^^ not sure :/